src/pages/login/login.js

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios'
+import { useEffect, useState } from 'react'
+import { useAuth } from '../../filter-context/auth-context'
+import './login.css'
+import { useNavigate } from 'react-router-dom'
+export const Login=()=>{
+    const navigate=useNavigate()
+    const {auth,setAuth}=useAuth()
+    const [show,setShow]=useState({email: "",
```

Comment:
name is not good. Should be something like `[userDetails,setUserDetails]`

Reply:
ok will correct it